core/src/test/scala/org/apache/spark/deploy/history/FsHistoryProviderSuite.scala

```diff
@@ -143,12 +156,26 @@ class FsHistoryProviderSuite extends SparkFunSuite with BeforeAndAfter with Matc
       SparkListenerApplicationStart("app1-2", Some("app1-2"), 1L, "test", None),
       SparkListenerApplicationEnd(2L)
       )
+
+    val path = new Path(logFile2.toURI)
+    val fs = path.getFileSystem(SparkHadoopUtil.get.conf)
+    val status = fs.getFileStatus(path)
+    SparkHadoopUtil.get.checkAccessPermission(status, FsAction.READ) should be (true)
```

Comment:
Why are these tests here? There's no `SparkHadoopUtilSuite` currently but it makes a lot more sense to create one, since that's where the code lives.

You also don't need to reference real files here; you can create custom `FileStatus` objects to test the behavior you want (the constructors are public and provide all the parameters you need).